tfjs-converter/python/tensorflowjs/converters/tf_saved_model_conversion_v2.py

```diff
@@ -127,6 +127,36 @@ def _run_grappler(config, graph_def, graph, signature_def):
   return tf_optimizer.OptimizeGraph(
       config, meta_graph, cluster=get_cluster())
 
+def get_output_node_names(node_map, target):
```

Comment:
Thanks Lin, it would be better to move this graph def manipulation to another file, please see fuse_prelu.py or fold_batch_norms.py as examples.
Also please add tests for this change.